tests/integration/relations/test_db_admin.py

```diff
@@ -22,14 +23,14 @@
 
 logger = logging.getLogger(__name__)
 
-FIRST_DISCOURSE_APP_NAME = "discourse-k8s"
```

Comment:
discourse-k8s was ported to the new interface.

Reply:
Crazy naming choise... BTW, is "discourse-k8s" R.I.P? Why do we need two identical charms? o_0